deps/rabbit/src/rabbit_vhost.erl

```diff
@@ -271,18 +271,23 @@ delete(VHost, ActingUser) ->
     %% modules, like `rabbit_amqqueue:delete_all_for_vhost(VHost)'. These new
     %% calls would be responsible for the atomicity, not this code.
     %% Clear the permissions first to prohibit new incoming connections when deleting a vhost
+    rabbit_log:info("Clearing permissions in vhost '~ts' because it's being deleted", [VHost]),
```

Comment:
I think most of these should be at debug level

Reply:
there're a lot of important elements being deleted whenever a vhost is deleted which we always need more log exposure to, without having to toggle log level to debug. we're seeing some [issues](https://github.com/rabbitmq/rabbitmq-server/pull/10320#issuecomment-1888805083) with incomplete vhost deletions.

Reply:
I don't have a problem with these messages specifically being logged at info level but the core team would like to avoid over-optimising RabbitMQ defaults for just one user who needs something.

Reply:
_"over-optimising RabbitMQ"_, I didn't know that was even a thing 😅 seeing we usually appreciate any/all performance gains. anyway, the other part of this pr was removing some unnecessary repeat vhost:exist checks. the bigger problem we want to expose here is with the vhost deletion logging. Just a couple problems we're trying of fix out here! cheers 👍

Reply:
Perhaps I should have said "over-fitting". I was talking about one specific part of this PR that affects all users but was suggested by just one.